### ISSUE TITLE
Improve error message on wrong project ID

### ISSUE THREAD

**Issue by invidian** [terraform-providers/terraform-provider-packet#146]
Currently, if specified project ID is incorrect and you try to create a device, Terraform only throws following message:
```
* packet_device.worker_nodes.0: Not found
```

Which is very misleading.

### Terraform Version
```
Terraform v0.11.13
+ provider.packet v1.7.2
```

### Affected Resource(s)
Please list the resources as a list, for example:
- packet_device

### Expected Behavior
Terraform should inform user, that project with given ID is not found.

### Actual Behavior
Terraform only returns "Not found" as error message.

**Comment by t0mk** [comment]
hey @invidian thanks for reporting this.

I am not sure if we can do sth about this, because the device creating POST is going to /projects/<project_id>/devices, and thus the 404 is a legit HTTP response.

But I think that the 404 only comes when the project doens't exist, so I can extend the err message a bit.

**Comment by t0mk** [comment]
it will be fixed by #147, I hope this can wait till next provider release
